image_classification/tensorflow/official/resnet/resnet_run_loop.py

```diff
@@ -401,12 +429,30 @@ def resnet_main(seed, flags, model_function, input_function, shape=None):
   else:
     benchmark_logger = None
 
-  for _ in range(flags.train_epochs // flags.epochs_between_evals):
+  mlperf_log.resnet_print(key=mlperf_log.TRAIN_LOOP)
+  success = False
+  for i in range(flags.train_epochs // flags.epochs_between_evals):
+    for j in range(flags.epochs_between_evals):
+      mlperf_log.resnet_print(key=mlperf_log.TRAIN_EPOCH,
```

Comment:
Clarify why you print multiple epoch numbers in a row here.

Reply:
Added.